cmd/minishift/cmd/delete.go

```diff
@@ -61,6 +60,7 @@ func runDelete(cmd *cobra.Command, args []string) {
 	}
 
 	if clearCache {
+		fmt.Println("Deleting Minishift cache")
```

Comment:
I think we need to be more verbose than that e.g. We should say what files are getting deleted or atleast which directories.